presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/TwoStatesMapping.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.state;
+
+import com.facebook.presto.spi.block.Block;
+import com.facebook.presto.spi.function.AccumulatorState;
+import io.airlift.slice.Slice;
+import javafx.util.Pair;
```

Comment:
Don't use this class from `javafx`. Use `ImmutableList.of(a, b)`

Usually, you should introduce a custom class with semantic meaning, though in this case, using a list as tuple is reasonable since there's no special semantic meaning to the items.

See http://stackoverflow.com/a/156685/17944